Dashboard/app/js/lib/views/maps/map-views-common-public.js

```diff
@@ -0,0 +1,205 @@
+FLOW.NavMapsView = FLOW.View.extend({
```

Comment:
I noticed this is declared twice and I was not sure whether or not that is accepted by the framework.